ansible-inventory-ocp-39-aio

```diff
@@ -0,0 +1,29 @@
+
+[OSEv3:children]
+nodes
+masters
+etcd
+
+[OSEv3:vars]
+ansible_connection=local
+openshift_release=v3.9
+openshift_logging_install_logging=true
+short_version=3.9
+openshift_deployment_type=openshift-enterprise
+openshift_logging_image_prefix=registry.access.redhat.com/openshift3/
+openshift_master_identity_providers=[{'mappingMethod': 'lookup', 'challenge': 'true', 'login': 'true', 'kind': 'AllowAllPasswordIdentityProvider', 'name': 'allow_all'}]
+openshift_logging_es_allow_external=True
+openshift_logging_use_mux=True
+openshift_logging_mux_allow_external=True
+openshift_logging_mux_file_buffer_storage_type=hostmount
+openshift_logging_elasticsearch_storage_type=hostmount
+openshift_logging_elasticsearch_hostmount_path=/var/lib/elasticsearch
+
+[nodes]
+localhost storage=True openshift_node_labels="{'region': 'infra'}" openshift_schedulable=True
+
+[masters]
+localhost storage=True openshift_node_labels="{'region': 'infra'}" openshift_schedulable=True
+
+[etcd]
```

Comment:
How will this affect the installation? Is it required in the oVirt use case?

Reply:
I see you removed these flags:
-openshift_hosted_logging_deploy=true
-openshift_logging_namespace=logging
-openshift_logging_es_cluster_size=1
-openshift_logging_image_version=v3.6
-deployment_type=openshift-enterprise
-openshift_disable_check="package_version,docker_storage"

Are they not needed anymore?

Reply:
> How will this affect the installation? Is it required in the oVirt use case?

not sure what you mean

Reply:
see latest